### DISCUSSION TITLE
Can I use the Ely by skin system?

### DISCUSSION THREAD

**Discussion by Magmaout** [PrismLauncher/PrismLauncher#979] [Q&A] [answered]
Ely by is a system of skins for minecraft https://ely.by/, my friends use it and I want to see their skins, but on PrismLauncher the mod is not visible in the list of mods, and there are no skins (the TLauncher skin system works, but I need Ely by)

**Comment by xllifi** [comment]
use https://modrinth.com/mod/customskinloader and configure it to use elyby first, then mojang

**Reply by askelane142** [reply]
how do i configure it? in 1.8.9, there is no config option

**Reply by gamfam1** [reply]
> how do i configure it? in 1.8.9, there is no config option

You can use https://github.com/fn2006/PollyMC, which is a fork of Prism with ely.by support.

**Reply by Anonymous941** [reply]
@gamfam1 You can just use regular PolyMC, they allow authlib-injector while still requiring a Microsoft account to not allow piracy.  PollyMC removes _all_ checks which means it directly allows piracy

**Reply by Kostya0Sim** [reply]
For a person who put a dislike reaction, use UltimMC.

**Reply by askelane142** [reply]
ultim mc sucks. elyprism is a real deal

**Comment by forkiesassds** [comment]
Prism Launcher does not support pirated copies of Minecraft or piracy in general.

Edit: turns out I was informed of this post just now because of a necropost???

**Reply by Anonymous941** [reply]
Why can't they allow authlib-injector as long as you have a Microsoft account?  That's not piracy, and it's what PolyMC already does.  The only alternatives are connecting to Microsoft constantly which is a privacy nightmare, or enabling offline mode which makes it possible for anyone to get OP

**Comment by ginqus** [comment]
https://github.com/Octol1ttle/ElyPrismLauncher

**Reply by Anonymous941** [reply]
https://github.com/unmojang/FjordLauncher is an alternative to that which does not allow piracy

**Comment by Lutitious** [comment, marked as answer]
Do not discuss piracy here

**Reply by Anonymous941** [reply]
I don't understand why this is piracy.  It's just an alternate to offline accounts that doesn't use Mojang's authentication system, not a way to get the game for free.  If I'm mistaken please correct me...

**Reply by Lutitious** [reply]
> I don't understand why this is piracy. It's just an alternate to offline accounts that doesn't use Mojang's authentication system, not a way to get the game for free. If I'm mistaken please correct me...

Because that provides a way to pirate accounts if you do not own the game. Again do not discuss piracy here
